lib/src/type/models/equality.dart

```diff
@@ -8,6 +8,18 @@ abstract class Equality<T> {
   /// Returns `true`, if [a] and [b] are the same.
   bool isEqual(T a, T b);
 
+  /// Returns `true`, if [a] is less than [b].
```

Comment:
Note, this is already implemented as part of `type.comparator.lessThan`, ...

Reply:
I removed the possible duplicates